packages/flutter_settings/lib/src/config/controls/toggle.dart

```diff
@@ -0,0 +1,29 @@
+import "package:flutter/material.dart";
+import "package:flutter_settings/flutter_settings.dart";
+import "package:settings_repository/settings_repository.dart";
+
+///
+class ToggleControlConfig
+    extends DescriptiveTitleControlConfig<bool, ToggleControlConfig> {
+  ///
+  ToggleControlConfig({
```

Comment:
this could all be const if you make SettingsControlConfig const and follow the classes that extend it